script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -86,83 +98,116 @@ I mean, even with a gag order, there's going to be leaks, right?"
- Young master, at your age, you're going on the fast track at full speed. Speaking of your pedigree and mentality. I mean, don't you feel the desire to act a bit more like the other kids your age and go have some fun?"
+// I think canon is that he's 'adopted', right? So lineage/pedigree for 家柄
+// probably isn't the best, it's also something that can't be actively
+// improved, so it's not like he can be dedicating himself to it instead of
+// more 年相応の遊び. Rendering as social standing instead.
+// Mario grindset
+ You're still young but you're already gunning it for the fast track aren't you, young master?
+ Both in terms of your social status and your mindset.
+ I mean, don't you ever want to act your age and play around once in a while?"
 }
 [sha:6b5a46592fca4924d9eda81d955c113f1bc5c10d]{
 -- Page 797, Offset 13011.
 -- 「あるかンなもん。ガキってのは基本つまらねえもんだって忘れたのか30歳？　オラ、無駄話してねえで監督してきやがれ。ヤバめな証拠は隠滅しとけよ」
 // He says 30歳, thirty year-old. Just imagine that I thought about "30 year old boomer" for a second haha
-"No fucking way. Have you forgotten that brats are all dull, you 30-year-old-geezer? Quit it with your nonsense and go supervise. And make sure to destroy any dangerous evidence."
+"Absofucking%{i}lutely%{/i} not.
+ Have you forgotten how goddamn dull being a kid is, you thirty-year-old geezer?
+ Come on, quit spouting crap and go keep and eye on those fuckers.
+ And if you come across any dangerous evidence, torch it."
 }
 [sha:e84e94817169e354e0a426bb913baf775ca1075f]{
 -- Page 797, Offset 13012.
 -- 「へーい。上司の命令とあればよろこんでー」
-"Yeah. If those are my boss's orders, with pleasure."
+"Alrighty. If those are my orders, then with pleasure."
 }
 [sha:a5be56357297b2f26e024694d5b19b0989a57747]{
 -- Page 798, Offset 13013.
 -- 「……アレでもちっと緊張感があれば使えるんだがな。あの捨て鉢さはなんだ、若い頃に挫折でもしたのかカリウス？」
-"...If only he had some sense of urgency, then he'd be more useful. To be so low-spirited, did he break down when he was young or something? What do you think, Karius?"
+"...That guy would really come in handy if he had any fucking sense of urgency at all.
+ How'd he end up in such a sorry state...
+ His spirit get crushed when he was young or something, Karius?"
 }
 [sha:c99b4827f10eb2502f08740495726839c3e10818]{
 -- Page 799, Offset 13014.
 -- 「私にはとんと。安藤の私生活に興味はありませんので。
-"I haven't the slightest idea. I don't have any interest in Andou's private life.
+"I haven't the slightest idea. Andou's private life holds no interest for me.
 }
 [sha:4d5e1406f183d7f3cc5b7de59cefb742bdfb0098]{
 -- Page 799, Offset 13015.
 -- ３年前、悪徳司祭の内部告発を決意したものの、最後の最後で金に目が眩んだ恋人に裏切られ、故郷を追われ、失意の果てに父方の祖国であるこの国に島流しになった、という事ぐらいしか。
-I only know that he decided to blow the whistle on a corrupt priest, but at the last minute, was betrayed by his lover who had been blinded by the promises of riches, was driven out from his hometown, and was, at the depths of despair, exiled to Japan, his paternal native country.
+All I know is that three years ago he decided to blow the whistle on a corrupt presbyter,
```

Comment:
I'm hesitant to add another comma to _the_ run-on sentence of the year, but
> years ago, he decided

Maybe just reflow it to something like:
> All I know is that three years ago, he decided to blow the whistle on a corrupt presbyter. He got betrayed in the final hour though by his lover, who had been blinded by greed. After that, he was driven from his hometown before, as the final act in his tale of adversity, he was exiled to his father's homeland―――to this country.

Reply:
I'm happy to add another comma, I think we should keep this sentence as heinously long as possible since it's basically a joke